pom.xml

```diff
@@ -92,7 +93,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
-                <version>3.1</version>
+                <version>3.8.1</version>
```

Comment:
could you please move this change to one more separate commit?

Reply:
You want two more PRs, right?

Reply:
it doesn't matter: it could 2 more commits in this PR or 2 more PRs
the reason is to keep the changes atomic

Reply:
done